src/tests/conftest.py

```diff
@@ -114,9 +57,11 @@ async def test_app_asyncio(__app):
 
 
 @pytest.fixture(scope="function")
-def test_db():
+async def test_db():
     try:
+        await test_database.connect()
         yield test_database
     finally:
         print("Clean the DB")
```

Comment:
Actually, we can remove this, it won't get printed

Reply:
It's printed in case there is a failing test.
Thought it was informative in case we wanted to be sure it's cleant.